src/code/z_lights.c

```diff
@@ -183,7 +183,7 @@ LightNode* Lights_FindBufSlot(void) {
 }
 
 // return type must not be void to match
-s32 Lights_FreeNode(LightNode* light) {
+BAD_RETURN(s32) Lights_FreeNode(LightNode* light) {
```

Comment:
in my opinion the comment can go now that BAD_RETURN documents the reason. I feel similarly about the comment in player for example, but have never gotten around to removing it myself